src/Sylius/Bundle/CoreBundle/Resources/config/app/sylius.yml

```diff
@@ -113,6 +113,7 @@ sylius_order:
         order_item_unit:
             classes:
                 model: Sylius\Component\Core\Model\OrderItemUnit
+    cart_expires_after: 3 hours
```

Comment:
True that. I'm for 2 days, as a default expiration period. What say you (you all)? 🗡